### PR TITLE
DIT-1023 - Cancel ruling with critical evidence

### PR DESCRIPTION
**Pull request by aaroncarroll82**
Draft PR for now as I haven't even looked at unit tests yet

### CODE REVIEW

File: app/uk/gov/hmrc/tariffclassificationfrontend/models/CancelRuling.scala [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tariffclassificationfrontend.models
+
+import uk.gov.hmrc.tariffclassificationfrontend.models.CancelReason.CancelReason
+
+case class CancelRuling(cancelReason: String, note: String)
```
Comment (ed0906) [inline app/uk/gov/hmrc/tariffclassificationfrontend/models/CancelRuling.scala]: maybe `RulingCancellation`?

File: app/uk/gov/hmrc/tariffclassificationfrontend/service/CasesService.scala [hunk 2]
```diff
@@ -35,6 +35,7 @@ import uk.gov.hmrc.tariffclassificationfrontend.models.request.NewEventRequest
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
+//noinspection ScalaStyle
```
Comment (ed0906) [inline app/uk/gov/hmrc/tariffclassificationfrontend/service/CasesService.scala]: Why are these appearing?

Comment (aaroncarroll82) [reply app/uk/gov/hmrc/tariffclassificationfrontend/service/CasesService.scala]: They won't be there in final version.

File: app/uk/gov/hmrc/tariffclassificationfrontend/models/CancelRuling.scala (hunk 1 above)
Comment (aaroncarroll82) [reply app/uk/gov/hmrc/tariffclassificationfrontend/models/CancelRuling.scala]: Yep think that is a better name, will use that.

**Pull request merged**